<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_autf</class>
 <widget class="QDialog" name="form_autf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	font-family: 'Montserrat', sans-serif;
	font-size: 14px;
	color: rgb(71, 74, 81);
}

QDialog{
	background-color: rgb(255, 255, 255);
}

QLineEdit{
	border: 1px solid rgb(0, 188, 188);
	border-radius: 3px;
}

QPushButton{
	height: 20px;
	font-family: 'Montserrat', sans-serif;
	font-size: 11px;
	text-transform: uppercase;
	font-weight: 600;
	color: rgb(205, 92, 92);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QPushButton:hover{
	height: 75px;
	border: 1px solid rgba(205, 92, 92, 1) ;
	border-radius: 5px;
	font-family: 'Montserrat', sans-serif;
	font-size: 11px;
	text-transform: uppercase;
	font-weight: 600;
	color: rgb(205, 92, 92);
	background-color: rgba(255, 255, 255, 0);
}

QPushButton:pressed{
	background-color: rgba(205, 92, 92, 1);
	color: white;
	border:2px solid;
	border: 1px solid rgba(205, 92, 92, 1) ;
	border-radius: 5px;
	font-family: 'Montserrat', sans-serif;

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Database server address</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_serv">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Database name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_base">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_login">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_pass">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
